rllib/algorithms/crr/torch/crr_torch_policy.py

```diff
@@ -389,14 +407,15 @@ def _compute_critic_loss(
         ).squeeze(-1)
 
         # compute the MSE loss for all q-functions
-        loss_q1 = (target - q1) ** 2
-        loss_q2 = (target - q2) ** 2
-        loss = 0.5 * (loss_q1 + loss_q2)
+        loss_q1 = target - q1
```

Comment:
since we are here, can we just unify the naming with DQN? loss_q1 --> td_error_q1, etc.
The sign will tell us about under or overfitting. and it should be q1 - target, similar to DQN's definition.

Reply:
done. let me know if you think the names are still not good.